FILES.deploy-tools/etc/ansible/ansible.cfg

```diff
@@ -15,7 +15,7 @@
 #library        = /usr/share/my_modules/
 remote_tmp     = $HOME/.ansible/tmp
 pattern        = *
-forks          = 1
+forks          = 15
```

Comment:
Is this a debug change? Are the Nebula folks ok with this number?

It may be possible to override this in the inventory, which might be preferable for large deploys.